app/services/forest_liana/search_query_builder.rb

```diff
@@ -50,7 +51,9 @@ def search_param
 
         @resource.columns.each_with_index do |column, index|
           column_name = format_column_name(@resource.table_name, column.name)
-          if column.name == 'id'
+          if (@collection.search_fields && !@collection.search_fields.index(column.name))
```

Comment:
`index` vs `include?` => include is really much more readable, more "ruby style"